code/modules/mob/living/simple_animal/friendly/snek.dm

```diff
@@ -0,0 +1,22 @@
+/mob/living/simple_animal/friendly/snek
+	name = "Snake"
+	desc = "Cute snek that needs a little boop"
```

Comment:
As much as it's a meme, I don't inherently have a problem with this - mostly because snakes are admin spawn only at the moment - but definitely put a period at the end.